### ISSUE TITLE
remove `aperv`

### ISSUE THREAD

**Issue by amereghe** [SixTrack/SixTrack#951]
This variable keeps track of the aperture at the loss point, and it is used only in `CR` and dump in `fort.6` of particle states. It stores only two numbers. Given the fact that the dump is not actually used (it is there only for user convenience) and the `apeture_losses.dat` stores the complete info, I propose to kill those variables.

**Comment by vkbo** [comment]
Yes, I agree. This was one of the variables I had to change the indexing of for the particle ID changes.
